### DISCUSSION TITLE
Ryujinx crashes

### DISCUSSION THREAD

**Discussion by gabbigamers** [jordanbtucker/totk-no-loot-boxes#4] [Q&A] [answered]
i cant open the game whenever i use this mod and im using ryujinx and also heres then mods im using: Unbreakable_Equipment, FOV_Mod_v0.1, Dualsense UI Mod v8 - Western Layout + Radial Menu  + Yuzu fix, Disable Dynamic Resolution, CameraSensitivity_Plus6_v2.

**Comment by jordanbtucker** [comment]
Does Ryujinx crash when you only have No Loot Boxes installed?

**Reply by gabbigamers** [reply]
Yes

**Reply by jordanbtucker** [reply]
- What version of Ryujinx are you using?
- Are you able to provide the logs from Ryujinx?

**Reply by lulanz** [reply]
i have the same problem

**Reply by jordanbtucker** [reply]
@lulanz Are you willing to provide the information I asked @gabbigamers for? I can't really troubleshoot problems with so little information.

**Reply by gabbigamers** [reply]
Ldn

**Reply by gabbigamers** [reply]
Ava

**Reply by gabbigamers** [reply]
> @lulanz Are you willing to provide the information I asked @gabbigamers for? I can't really troubleshoot problems with so little information.

Ava ldn

**Reply by gabbigamers** [reply]
> * What version of Ryujinx are you using?
> * Are you able to provide the logs from Ryujinx?

I am not able to provide logs but you would probably get the same error.

**Reply by lulanz** [reply]
[Ryujinx_1.1.808_2023-05-19_23-38-55.log](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11520694/Ryujinx_1.1.808_2023-05-19_23-38-55.log)

**Reply by jordanbtucker** [reply]
@gabbigamers You might be running TotK v1.0.0 while v1.1.0 or v1.1.1 is required for this mod. See https://github.com/jordanbtucker/totk-no-loot-boxes/discussions/4#discussioncomment-5953177

**Reply by gabbigamers** [reply]
> @gabbigamers You might be running TotK v1.0.0 while v1.1.0 or v1.1.1 is required for this mod. See [#4 (reply in thread)](https://github.com/jordanbtucker/totk-no-loot-boxes/discussions/4#discussioncomment-5953177)

I am running v1.1.0 actually

**Reply by gabbigamers** [reply]
> @gabbigamers You might be running TotK v1.0.0 while v1.1.0 or v1.1.1 is required for this mod. See [#4 (reply in thread)](https://github.com/jordanbtucker/totk-no-loot-boxes/discussions/4#discussioncomment-5953177)

oops sorry for saying i wasnt on 1.0.0 i have a bug in my ryujinx where version fallback happens the first time i launch the game lol

**Reply by jordanbtucker** [reply]
So is this problem fixed?

**Reply by gabbigamers** [reply]
Yeah but, it would be awesome if you made a mod that would make a new setting for adjusting fps and graphics and resolution then you would probably the most beloved mod developer lol

**Comment by lulanz** [comment]
[Ryujinx_1.1.808_2023-05-19_23-38-55.log](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11520687/Ryujinx_1.1.808_2023-05-19_23-38-55.log)

**Reply by lulanz** [reply]
i am new to ryujinx so idk if this is what you want

**Reply by jordanbtucker** [reply, marked as answer]
No worries. I see your problem is that you're running TotK v1.0.0 while the mod requires TotK v1.1.0 or v1.1.1.

Since you're using the XCI, do you have the v1.1.0 update NSP? If so:

- Open Ryujinx.
- Right-click Tears of the Kingdom > Manage Title Updates.
- Click Add.
- Select the TotK v1.1.0 update NSP file.
- Make sure Version 1.1.0 is checked.
- Click Save.

Now your game should work with the mod.

**Reply by lulanz** [reply]
wow this really worked 

sorry for being dumb xD

**Comment by DedZ3d** [comment]
I'm having the same problem. I've tried everything I can think of. Here's my log file. 
[Ryujinx_1.1.819_2023-05-23_21-11-32.log](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11549797/Ryujinx_1.1.819_2023-05-23_21-11-32.log)

**Reply by jordanbtucker** [reply]
It looks like you have two mods enabled. Please disable the other mod, try the game again, and post your new logs if it's still not working. Thanks.

**Reply by DedZ3d** [reply]
[Ryujinx_1.1.819_2023-05-24_10-24-03.log](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11556340/Ryujinx_1.1.819_2023-05-24_10-24-03.log)
